### DISCUSSION TITLE
office文件无法预览

### DISCUSSION THREAD

**Discussion by xuyuanyyds123** [AlistGo/alist#2866] [Q&A] [answered]
咨询office文件预览相关问题：
我挂载了多个储存，分别是对象存储、onedrive和天翼云盘客户端，测试如下：

对象存储下Microsoft和Google可以预览，onlyoffice无法预览
![image](https://user-images.githubusercontent.com/109422817/210167190-7a18430e-dac8-4665-a696-8b2c69495470.png)
![image](https://user-images.githubusercontent.com/109422817/210167232-8b0d2bc0-6f6a-42ce-a1f6-0d611f263bd0.png)

onedrive和天翼云盘客户端下只能使用Microsoft，Google和onlyoffice都无法使用
![image](https://user-images.githubusercontent.com/109422817/210167330-0928dcfb-869b-4452-ab3f-90bccdd1994d.png)
![image](https://user-images.githubusercontent.com/109422817/210167347-cf9e113c-63c0-42b4-80c8-7808f971d01c.png)

**运行环境：winserver2022，百度云服务器，
alist3.7.2，HTTPS访问**
该问题已经困扰我很久了，一直都是这样，已经排除onlyoffice配置不正确的可能

**Comment by NovaShen0724** [comment, marked as answer]
1 google可以打开
<img width="960" alt="2023 1 3" src="https://user-images.githubusercontent.com/46617265/210390791-444472dc-ab86-45d6-b706-a375c9ac74e1.png">
2 您的nginx是不是没有代理80端口
3 您的nginx是不是没有设置传输编码，使用edge浏览器发现您网站下方文字为乱码
4 请提供fynn的接口的配置文件，就是那个框里的代码都复制出来
5 Personal question，您是哪个省的OIer呀

**Reply by xuyuanyyds123** [reply]
1，欧克我谷歌也好啦
2，其实我没用nginx呜呜呜呜，我我我我现在就去学嗷
3，我用的是那个，，那个，就是alist里面那个config，功能少很多但是方便（），那既然是这样，感谢大佬的提醒，我今天一定把nginx配置好！
4，OnlyOffice在线预览/编辑文档
请输入参数src 示例:
预览模式-> https://office.fynn.vip/view.html?src={$src}
编辑模式-> https://office.fynn.vip/edit.html?src={$src}
请求协议http和https都支持，请根据自己的网站设置同源协议
src参数配置请根据自己的网站参数配置配置接口 {$src}是网站提供的变量
5，山东淄博，一个高中，叫淄博七中（但我不算是Oler，因为我很菜呜呜呜呜呜）

**Reply by NovaShen0724** [reply]
2/3.nginx还是很有用滴
4.根据官方文档，应填写 [https://office.fynn.vip/view.html?src=$e_url](https://office.fynn.vip/view.html?src=$e_url) 
若版本不同可参照默认配置的参数，就是{$src}那要替换掉
5.大佬大佬，膜拜一下强省大佬

**Reply by zhushen12580** [reply]
> 1，欧克我谷歌也好啦 2，其实我没用nginx呜呜呜呜，我我我我现在就去学嗷 3，我用的是那个，，那个，就是alist里面那个config，功能少很多但是方便（），那既然是这样，感谢大佬的提醒，我今天一定把nginx配置好！ 4，OnlyOffice在线预览/编辑文档 请输入参数src 示例: 预览模式-> [https://office.fynn.vip/view.html?src={$src}](https://office.fynn.vip/view.html?src=%7B$src%7D) 编辑模式-> [https://office.fynn.vip/edit.html?src={$src}](https://office.fynn.vip/edit.html?src=%7B$src%7D) 请求协议http和https都支持，请根据自己的网站设置同源协议 src参数配置请根据自己的网站参数配置配置接口 {$src}是网站提供的变量 5，山东淄博，一个高中，叫淄博七中（但我不算是Oler，因为我很菜呜呜呜呜呜）

大佬 edit.html可以发一下吗
